app/forms/state_file/md_two_income_subtractions_form.rb

```diff
@@ -0,0 +1,24 @@
+module StateFile
+  class MdTwoIncomeSubtractionsForm < QuestionsForm
+    set_attributes_for :intake, :primary_student_loan_interest_ded_amount, :spouse_student_loan_interest_ded_amount
+
+    validate :valid_amounts
+
+    def save
+      @intake.update(attributes_for(:intake))
+    end
+
+    private
+
+    def valid_amounts
+      if @intake.direct_file_data.fed_student_loan_interest.present?
+        if (primary_student_loan_interest_ded_amount.to_f + spouse_student_loan_interest_ded_amount.to_f).to_i != @intake.direct_file_data.fed_student_loan_interest.to_i
```

Comment:
was there a reason we chose `to_i` over `to_round`? 
If a client enters 5.25 and 5.25 for both amounts, and the resulting sum is 10.5, is it more correct to make sure it gets compared to the `federal_student_loan_interest` as 10 or 11? I'm guessing this is a really unlikely situation here that someone seeing an integer value will purposefully add decimal points here.

Reply:
hmm this should call `@intake.direct_file_data.fed_student_loan_interest.to_f` instead then, and the prev value should not call to_i, so that the decimals are compared properly